packages/eos-components/src/components/ButtonsBar.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div>
+    <b-button
+      v-for="button in buttonsVisible"
+      :key="button"
+      pill
+      size="sm"
+      variant="primary"
+      class="ml-2"
+      @click="$emit('click', button)"
+    >
+      {{ button }}
+    </b-button>
+    <b-dropdown
+      v-if="buttonsHidden.length"
+      ref="dropdown"
+      size="sm"
+      class="ml-2"
+      variant="primary"
+      text="···"
+      noCaret
```

Comment:
It's better to aling this dropdown to the right, or it can appear outside the screen:
![Captura de pantalla de 2021-07-06 18-37-58](https://user-images.githubusercontent.com/83944/124670128-e93e6b80-de89-11eb-8fc8-ae290b92ccfb.png)

```suggestion
      noCaret
      right
```